src/main/java/com/learning/pages/google/GoogleSearchPage.java

```diff
@@ -0,0 +1,62 @@
+package com.learning.pages.google;
+
+import com.learning.configuration.PropertiesReader;
+import com.learning.pages.PageBase;
+import java.util.Objects;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class GoogleSearchPage extends PageBase {
+
+  @FindBy(name = "q")
+  private WebElement searchBox;
+  @FindBy(name = "btnK")
+  private WebElement searchButton;
+
+  private final PropertiesReader propertiesReader;
+  private static final String GOOGLE_URL_PROPERTIES_NAME = "googleURL";
+
+  public GoogleSearchPage(WebDriver driver, PropertiesReader propertiesReader) {
+    super(driver);
+    this.propertiesReader = propertiesReader;
+  }
+
+  public void openGoogleWebsite(){
```

Comment:
for simplicity, this method could be called `open()`, no need to state same thing twice: `googleSearchPage.open()` instead of `googleSearchPage.openGoogleWebsite()`